code-conventions.md

```diff
@@ -165,15 +173,23 @@ Your comments should look something like these:
   - Users must not need JRE newer than version 8
 * Versions up to and including Java 17 should work too
   - Please avoid using unsafe reflection
+
+### Best Practices
 * It is recommended to make fields final, if they are effectively final
 * Local variables and method parameters should not be declared final
 * Methods should be declared final only where necessary
 * Use `@Override` whenever applicable
+* Avoid initializing multiple fields on one line like so:
+  ```java
+  long a = 0, b = 0; // bad
+  
+  // good
+  long a = 0;
+  long b = 0;
+  ```
```

Comment:
I don't agree with this, this can be really annoying when you've a bunch of fields that has the same modifiers, even for few ones, it's just better to merge them together

Reply:
Discussion continued on [discord](https://discord.com/channels/135877399391764480/836220422223036467/1070080468743098398)

Result: No problem, but preferrably allow some usage of it such as local variables if it doesn't hurt